src/database/resolvers.js

```diff
@@ -36,8 +38,21 @@ exports.resolver = {
 				.sort({ publishedDate: -1 })
 				.limit(100)
 
-			//   const sortedTweets = getSortedTweets(tweets);
 			return tweets
 		},
 	},
+	Mutation: {
+		storeFcmToken: async (parent, args) => {
+			const {
+				input: { fcmToken, countryCode },
+			} = args
+			console.log('_______________fcmtoken_______________', fcmToken)
```

Comment:
@shrijan-siris you planning to remove `console.log` from final pr? We should start using `winston` for logging.

Reply:
not for now we may remove after using winston

Reply:
yeah, with `winston`, we should be able to enable console logging only during development and also have different log levels like debug, info, warn per environment (dev, prod)